src/components/RecordSection.vue

```diff
@@ -20,29 +26,34 @@ export default defineComponent({
     currVisit: Boolean,
     date: String
   },
-  data() {
-    return {
-      dropdownOpen: false
-    }
-  },
   computed: {
     formattedDate(): string | null {
-      if (!this.date) return null;
-      const date = new Date(this.date);
-      const day = String(date.getDate()).padStart(2, '0');
-      const month = String(date.getMonth() + 1).padStart(2, '0'); // getMonth() returns month index starting from 0
-      const year = date.getFullYear();
-      return `${day}-${month}-${year}`;
+      if (!this.date) return null
+      const date = new Date(this.date)
+      const day = String(date.getDate()).padStart(2, '0')
+      const month = String(date.getMonth() + 1).padStart(2, '0') // getMonth() returns month index starting from 0
+      const year = date.getFullYear()
+      return `${day}-${month}-${year}`
     }
   },
   methods: {
-    toggleDropdown() {
-      this.dropdownOpen = !this.dropdownOpen
-    },
     handleClick() {
-      this.$router.push({ name: 'patient', params: { id: this.id, vid: this.vid } });
+      this.$router.push({ name: 'patient', params: { id: this.id, vid: this.vid } })
       this.$emit('close')
     }
   }
 })
-</script>
\ No newline at end of file
+</script>
```

Comment:
Why is there custom button css? Doesn't this undo the records section button highlighting